### PR TITLE
Reflecting renaming changes on EventFlow code base

### PR DESCRIPTION
**Pull request by matzew**
needs to wait until this is in:

https://github.com/project-streamzi/cloudevent-flow/pull/68

### CODE REVIEW

File: roles/provision-event-flow-apb/tasks/provision-manager.yml [hunk 1]
```diff
@@ -59,7 +59,7 @@
 
 - name: Add CRD for Flows
   when: flows_result.rc != 0
-  command: "kubectl create -f https://raw.githubusercontent.com/project-streamzi/cloudevent-flow/master/manager/src/main/resources/flow-crd.yml"
+  command: "kubectl create -f https://raw.githubusercontent.com/project-streamzi/cloudeventflow/master/ui/src/main/resources/flow-crd.yml"
```
Comment (sjwoodman) [inline roles/provision-event-flow-apb/tasks/provision-manager.yml]: When we move the repo we should rename the project to be 'eventflow'

**Review by sjwoodman — APPROVED** [review]
LGTM

File: roles/provision-event-flow-apb/tasks/provision-manager.yml (hunk 1 above)
Comment (matzew) [reply roles/provision-event-flow-apb/tasks/provision-manager.yml]: yes, that's planed _after_ this

Comment (matzew) [reply roles/provision-event-flow-apb/tasks/provision-manager.yml]: actually _BEFORE_ this - and after the changes on the actual source :grinning:

**Comment by matzew** [comment]
@sjwoodman so... we should rename this too :-)

`eventflow-apb` ... Will do ! :-)

**Pull request merged**